src/components/touchables/TouchableHighlight.tsx

```diff
@@ -61,21 +69,23 @@ export default class TouchableHighlight extends Component {
       extraChildStyle: null,
       extraUnderlayStyle: null,
     });
-    this.props.onHideUnderlay && this.props.onHideUnderlay();
+    this.props.onHideUnderlay?.();
   };
 
   renderChildren() {
     if (!this.props.children) {
       return <View />;
     }
 
-    const child = React.Children.only(this.props.children);
+    const child = React.Children.only(
+      this.props.children
+    ) as React.ReactElement<ViewProps>; // TODO: not sure if OK but fixes error
```

Comment:
Type assertion.

Reply:
`children` is `React.ReactNode` by default. Maybe we should type `children` explicitly to be `React.ReactElement<ViewProps>` or maybe something else?